sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -130,10 +130,10 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
   }
 
   test("show tblproperties for datasource table - errors") {
-    val message1 = intercept[NoSuchTableException] {
+    val message = intercept[AnalysisException] {
```

Comment:
Ur, this sounds like another behavior change from Apache Spark 2.4.4.
cc @gatorsmile

Reply:
Thanks for catching it! I will update the migration guide.

Reply:
Created a follow-up PR: https://github.com/apache/spark/pull/27276

Reply:
Thanks!